### PR TITLE
start/stop option for COVAREP_feature_formant_extraction_perfile

### PR DESCRIPTION
**Pull request by twoertwein**
This allows to extract features from only a part of an audio file (timestamps are adjusted accordingly).

I also included a missing dependency: NAQ, QOQ, ... depend on the pitch feature.

### CODE REVIEW

File: feature_extraction/COVAREP_feature_formant_extraction_perfile.m [hunk 1]
```diff
@@ -291,6 +307,9 @@
     results = [results array2table(signal, 'VariableNames', names)];
 end
 
+%% adjust time to account for different start point
+results.time = results.time + options.start;
```
Comment (gillesdegottex) [inline feature_extraction/COVAREP_feature_formant_extraction_perfile.m]: options.start has to be updated according to start_index before added to results.time.
options.start = (start_index-1)/fs % I think

Comment (twoertwein) [reply feature_extraction/COVAREP_feature_formant_extraction_perfile.m]: good point! `options.start` might not be the same as `(start_index-1)/fs` if there have been rounding issues. I will push a change in a few minutes

Comment (gillesdegottex) [reply feature_extraction/COVAREP_feature_formant_extraction_perfile.m]: Exact. Great thanks!

**Comment by twoertwein** [comment]
ups, I forgot to add a `;` in the last commit. Can you please add the `;` here https://github.com/covarep/covarep/blob/1bca05950c4aac54677a0e9a9315fdd27dbc5864/feature_extraction/COVAREP_feature_formant_extraction_perfile.m#L94

**Pull request merged**